### ISSUE TITLE
MRK 14 71

### ISSUE THREAD

**Issue by kim-putter** [Door43/udb-en#84]
But he began to call down upon God to punish him if he was not telling the truth; he said, "I do not know the man that you are talking about!"

The phrasing "he began to call down upon God to punish him" is a bit awkward. Perhaps: "he began to call down curses upon himself, saying for God to punish him"

**Comment by hmw3** [comment]
I unpacked "call down curses." Now reads:

\v 71 But he began to say that God could punish him if he was not telling the truth; he said, "I do not know the man that you are talking about!"

I would like to say "if he **_were_** not telling," but I think the subjunctive is beyond most ESL readers.

**Comment by pjoakes** [comment]
Henry,

The principle that we should follow is to use correct English, even though the typical American on the street may not. The subjunctive is quite common among other languages, not the strange, not-much-used thing that it is to Americans. Also, the people using the UDB (gateway language translators) are well-educated and would certainly know that this is incorrect. That leaves us open to criticism, and introduces the idea that, if the language is wrong (sloppy), how trustworthy is the translation?

Perry Oakes
perry_oakes@wycliffeassociates.org<mailto:perry_oakes@wycliffeassociates.org>



On Aug 12, 2016, at 8:27 AM, Henry Whitney <notifications@github.com<mailto:notifications@github.com>> wrote:


I unpacked "call down curses." Now reads:

\v 71 But he began to say that God could punish him if he was not telling the truth; he said, "I do not know the man that you are talking about!"

I would like to say "if he were not telling," but I think the subjunctive is beyond most ESL readers.

**Comment by hmw3** [comment]
Hi, Perry,


First and last, I'm happy that you want to use the subjunctive, and I'll make sure it's in there.


However, the idea of "correct English" is a bit of a problem. I grew up an "English Pharisee" -- "lie" is always intransitive, "lay" is always transitive (except when it's the past tense of "lie"), the whole nine yards. (I even trained my wife on when to use "lay" and when to use "lie.") My first Chicago Manual of Style delighted me: hundreds and hundreds of pages of rules about how things are supposed to be.


Except they're not. The serial comma is the one I fight most: Mrs. Piispanen told me in second grade that it's "Abercrombie [actually, she said, 'Ambercrombie'], Benjamin, and Christopher," not "Abercrombie, Benjamin and Christopher." Imagine my horror when I opened the New International Version and realized they'd left out all those commas! Actually, most publications seem not to use it.


And in Singapore, I've heard, you buy "a pant," not "a pair of pants." Which brings up the fact, at least I think it's a fact, that most people who use English every day speak it as a second language.


I have been editing academic papers written by non-native English speakers for 13 years. These are graduate students in theology and engineering. Very rarely will I see the subjunctive used at all, let alone incorrectly, let even more alone correctly. When I replace an indicative with a subjunctive, I feel like I need to explain it. Maybe they won't come back and say, "Why is it 'I were' instead of 'I was'?" but I figure I'd best make sure I beat them to it. Most of my customers speak languages (Korean, African languages) that as far as I can tell mark what we call subjunctivity in a way different from ours. I have spent a couple of months in Africa, and if I heard any African use the subjunctive, it went past me. So the difference between "I wish I were you" and "I wish I was you" is likely lost on the typical African ESL speaker, who is otherwise able to understand that "I" is sorry not to be "you."


Whatever "correct" English is, we have a standard for this project, and I'm happy that the subjunctive is part of it (as is the serial comma! [😊]  ). I think, however, it would be wise to speak in terms of "our standard" or "not our practice" rather than "correct" or "incorrect," even in house.


Now to make sure the subjunctive is in there!


Henry

________________________________

**Comment by hmw3** [comment]
Subjunctive clause made subjunctive.

**Comment by pjoakes** [comment]
Thanks, Henry. I suppose I’m something of an English Pharisee myself… I’ve been quite disappointed with the usage choices of the TNIV. The TNIV translators said they took as their standard the usage as recorded in some huge media database that compiled the English usage in American magazines and newspapers, which, as you know, are horrible in their command of English grammar. I think the Bible should be better.

Perry Oakes
perry_oakes@wycliffeassociates.org<mailto:perry_oakes@wycliffeassociates.org>



On Aug 12, 2016, at 11:34 AM, Henry Whitney <notifications@github.com<mailto:notifications@github.com>> wrote:

Hi, Perry,


First and last, I'm happy that you want to use the subjunctive, and I'll make sure it's in there.


However, the idea of "correct English" is a bit of a problem. I grew up an "English Pharisee" -- "lie" is always intransitive, "lay" is always transitive (except when it's the past tense of "lie"), the whole nine yards. (I even trained my wife on when to use "lay" and when to use "lie.") My first Chicago Manual of Style delighted me: hundreds and hundreds of pages of rules about how things are supposed to be.


Except they're not. The serial comma is the one I fight most: Mrs. Piispanen told me in second grade that it's "Abercrombie [actually, she said, 'Ambercrombie'], Benjamin, and Christopher," not "Abercrombie, Benjamin and Christopher." Imagine my horror when I opened the New International Version and realized they'd left out all those commas! Actually, most publications seem not to use it.


And in Singapore, I've heard, you buy "a pant," not "a pair of pants." Which brings up the fact, at least I think it's a fact, that most people who use English every day speak it as a second language.


I have been editing academic papers written by non-native English speakers for 13 years. These are graduate students in theology and engineering. Very rarely will I see the subjunctive used at all, let alone incorrectly, let even more alone correctly. When I replace an indicative with a subjunctive, I feel like I need to explain it. Maybe they won't come back and say, "Why is it 'I were' instead of 'I was'?" but I figure I'd best make sure I beat them to it. Most of my customers speak languages (Korean, African languages) that as far as I can tell mark what we call subjunctivity in a way different from ours. I have spent a couple of months in Africa, and if I heard any African use the subjunctive, it went past me. So the difference between "I wish I were you" and "I wish I was you" is likely lost on the typical African ESL speaker, who is otherwise able to understand that "I" is sorry not to be "you."


Whatever "correct" English is, we have a standard for this project, and I'm happy that the subjunctive is part of it (as is the serial comma! [😊] ). I think, however, it would be wise to speak in terms of "our standard" or "not our practice" rather than "correct" or "incorrect," even in house.


Now to make sure the subjunctive is in there!


Henry

________________________________
